ui/components/ui/editable-label/editable-label.js

```diff
@@ -12,8 +12,18 @@ class EditableLabel extends Component {
   state = {
     isEditing: false,
     value: this.props.defaultValue || '',
+    ensName: '',
   };
 
+  UNSAFE_componentWillReceiveProps(nextProps) {
```

Comment:
[Q] can you explain why these changes are necessary? We should really try to avoid adding more usages of UNSAFE lifecycle methods.

Reply:
If user has ENS, this is to prevent ENS from always being the default value whenever user clicks the edit button. But I am not going to do this way:) 
thanks